core/src/stepper/runge_kutta_dopri_stepper.cc

```diff
@@ -36,44 +36,45 @@ namespace jiminy
         // Evaluate error between both states to adjust step
         solution.difference(otherSolution_, error_);
 
-        // Compute absolute and relative element-wise maximum error
-        double errorAbsNorm = INF;
-        double errorRelNorm = INF;
-        if (tolAbs_ > EPS)
-        {
-            errorAbsNorm = error_.normInf() / tolAbs_;
-        }
-        if (tolRel_ > EPS)
-        {
-            otherSolution_.setZero();
-            solution.difference(otherSolution_, scale_);
-            error_ /= scale_;
-            errorRelNorm = error_.normInf() / tolRel_;
-        }
+        // Compute error scale
+        otherSolution_.setZero();
+        solution.difference(otherSolution_, scale_);
```

Comment:
It would be more robust to take the initial state as reference, so that the scale does not depend on the integration itself.
```
initialState.difference(otherSolution_, scale_);
```